core/src/main/scala/org/apache/spark/util/io/ChunkedByteBuffer.scala

```diff
@@ -40,6 +40,8 @@ private[spark] class ChunkedByteBuffer(var chunks: Array[ByteBuffer]) {
   require(chunks != null, "chunks must not be null")
   require(chunks.forall(_.position() == 0), "chunks' positions must be 0")
 
+  private val NIO_BUFFER_LIMIT = 64 * 1024 * 1024 // Chunk size in bytes
```

Comment:
actually i did not tested too much, it depends on value from our cluster.And i do some simple test then found 64mb will not affect perfomance compared with logic before.And benifit is we only need one buffer cache for block larger than 64 mb